### ISSUE TITLE
[BR-CO-17] category tax amount calculation

### ISSUE THREAD

**Issue by Lorenzschaef** [itplr-kosit/validator-configuration-xrechnung#78]
I get this error, even though, as far as I can see, the amounts are correct.

```
[BR-CO-17]-VAT category tax amount (BT-117) = VAT category taxable amount (BT-116) x (VAT category rate (BT-119) / 100), rounded to two decimals.
```

The same document with positive Amounts does not produce the error. I think it might be a bug in the validation rules?

<img width="882" alt="Bildschirmfoto 2022-07-07 um 09 47 41" src="https://user-images.githubusercontent.com/15876496/177720396-83e3b8d8-7b65-4145-983f-ec91dfebd0d9.png">

<img width="742" alt="Bildschirmfoto 2022-07-07 um 09 47 30" src="https://user-images.githubusercontent.com/15876496/177720403-28fae4bc-eda1-476c-a719-5a2b28d77ee1.png">

**Comment by phax** [comment]
Dear @Lorenzschaef, this is a very unfortunate situation.
The specific problem has already been addressed in CEN 1.3.8 rules, but the latest XRechnung release (2.2.0) relies on the CEN 1.3.7 rules. Due to backwards incompatible changes between 1.3.7 and 1.3.8 we cannot simply upgrade to the latest version.
The latest version of the CEN rules will be used with the next XRechnung Release.
Sorry for the bad news.

If you dare to, you might simply use the CEN 1.3.8 rules from https://github.com/ConnectingEurope/eInvoicing-EN16931/releases  in your validator configuration. But it might have unintended side effects, for which we cannot give any support.

**Comment by bdewein** [comment]
should be resolved with XRechnung 2.3 / update to CEN 1.3.9
